pkg/sql/parser/testdata/create_function

```diff
@@ -12,11 +12,11 @@ CREATE OR REPLACE FUNCTION f(IN a INT8 DEFAULT (7))
 CREATE OR REPLACE FUNCTION f(IN a INT8 DEFAULT _)
 	RETURNS INT8
 	LANGUAGE SQL
-	AS $$SELECT 1$$ -- literals removed
```

Comment:
ah just realized, this may be incorrect double quoting. we need to wrap with either `$$` or `'`', but not both.

Reply:
Good catch! Fixed